tests/core/process.c

```diff
@@ -293,6 +302,120 @@ TESTCASE (linux_process_modules)
   dlclose (lib);
 }
 
+TESTCASE (linux_get_cpu_from_auxv_null_32bit)
+{
+  GumCpuType cpu32;
+#if defined (HAVE_I386)
+  cpu32 = GUM_CPU_IA32;
+#elif defined (HAVE_ARM) || defined (HAVE_ARM64)
+  cpu32 = GUM_CPU_ARM;
+#elif defined (HAVE_MIPS)
+  cpu32 = GUM_CPU_MIPS;
+#else
+# error Unsupported architecture
+#endif
+  guint32 test_data[] =
+  {
+    AT_NULL, 0
+  };
+  g_assert_cmpuint (gum_linux_cpu_type_from_auxv ((const guint8 *)test_data,
+      sizeof (test_data)), ==, cpu32);
```

Comment:
```suggestion
  g_assert_cmpuint (gum_linux_cpu_type_from_auxv (test_data,
      sizeof (test_data)), ==, cpu32);
```

Reply:
Was missing a space after cast, but with the change to `gconstpointer` there's no need for a cast anymore.

(And same changes below.)